functional => main

4tinley opened the pull request:
Final version of the functional

Commit:
version_0.2

Just for initialize

Commit:
gameplay and tech => missing Visuals

Gameplay is done,
Tech is near to be like,
Visuals missing.

Commit:
26_09_end_of_day_version

Commit:
func file v2.1

Commit:
saved func file v2.1

Commit:
func file v2.-2

Commit:
first non checked final version

Commit:
git ignore added

Commit:
4docs images

Commit:
functional first review by all of us

Commit:
rectification of the previous commit

Commit:
next from rectification

Commit:
post final version verified (PGM only)

Commit:
cleaned branch

Commit:
final version => visuals missing

Commit:
another review

Commit:
version 1.0

Commit:
cleaned brach

Commit:
version controled

Commit:
Merge branch 'main' into functional

Enonome reviewed (changes requested):
For section 4.4, 1st question, maybe you wanted to mean "What are the winning conditions?" instead of "What are the win conditions?" Also, is it normal that the raccoon image does not appear in the functional_spec file?

GuillaumeDespaux committed:
Document order redone

The files where not at the good emplacement, so I moved them correctly in the tree.

GuillaumeDespaux committed:
Functional image fix

GuillaumeDespaux reviewed (approved):
Ok for me
